packages/manager/modules/billing-components/src/components/resiliation/resiliation.controller.js

```diff
@@ -5,15 +5,18 @@ export default class BillingResiliationController {
     this.BillingService = BillingService;
   }
 
-  resiliate() {
+  resiliate(endStrategy) {
     this.isResiliating = true;
 
     return this.BillingService.putEndRuleStrategy(
       this.service.id,
       this.endStrategy,
     )
       .then(() =>
-        this.onSuccess(this.$translate.instant('billing_resiliation_success')),
+        this.onSuccess(
+          this.$translate.instant('billing_resiliation_success'),
+          endStrategy,
```

Comment:
```suggestion
          this.endStrategy,
```
?

Reply:
endStrategy, it's comming from function parameter

Reply:
What's the difference with https://github.com/ovh/manager/pull/5413/files#diff-790a758b1dc215f30e7def61a6d6c832113e0cc8a2c7fba113031a979eeb2135R13 that is controller property ?